go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/fnproject/cli
 require (
 	github.com/coreos/go-semver v0.3.0
 	github.com/fatih/color v0.0.0-20170926111411-5df930a27be2
-	github.com/fnproject/fn_go v0.8.2
+	github.com/fnproject/fn_go v0.8.3-0.20210304131814-513b92cbf02a
```

Comment:
Might be worth creating a 'release' in fn_go at v0.8.3 and letting at the tag to the repo. Then we can just specify the tag here.

Reply:
Cheers, that's done 👍